gallery/plot_datapoints.py

```diff
@@ -0,0 +1,133 @@
+"""
+==============
+Datapoints FAQ
+==============
+
+The :mod:`torchvision.datapoints` namespace was introduced to ``torchvision.transforms.v2``. This example showcases what
+these datapoints are and how they behave. This is a fairly low-level topic that most users will not need to worry about:
+you do not need to understand the internals of datapoints to efficiently rely on ``torchvision.transforms.v2``. It may
+however be useful for advanced users trying to implement their own datasets, transforms, or work directly with the
+datapoints.
+"""
+
+import PIL.Image
+
+import torch
+import torchvision
+
+# We are using BETA APIs, so we deactivate the associated warning, thereby acknowledging that
+# some APIs may slightly change in the future
+torchvision.disable_beta_transforms_warning()
+
+from torchvision import datapoints
+
+
+########################################################################################################################
+# What are datapoints?
+# --------------------
+#
+# Datapoints are zero-copy tensor subclasses:
+
+tensor = torch.rand(3, 256, 256)
+image = datapoints.Image(tensor)
+
+assert isinstance(image, torch.Tensor)
+assert image.data_ptr() == tensor.data_ptr()
+
+
+########################################################################################################################
+# Under the hood, they are needed in :mod:`torchvision.transforms.v2` to correctly dispatch to the appropriate low-level
```

Comment:
I'm not sure that everyone here will understand what means "low-level kernels" here. Maybe, low-level transform functions or something less specific ? Especially that the word "kernels" in general have other popular meanings as CUDA kernels etc. This may lead to a certain confusion.

Reply:
I've switched to

> to correctly dispatch to the appropriate function for the input data.